testing branch

debbiecoder opened the pull request:
testing a pull request

debbiecoder committed:
testing branch

dmcs9296 commented on Brackets/Brackets/main.swift:
```diff
@@ -16,7 +16,7 @@ if let expression = readLine()
-var lenBrackets = inputBrackets.count
+let lenBrackets = inputBrackets.count
 var yesesAndNos: [String] = []
 
```
Do you need allBrackets ? Instead you can check if it is part of openBrackets or closedBrackets

dmcs9296 reviewed (commented):
Update to remove allBrackets

debbiecoder committed:
Updating Review Comment

Removed allBrackets, made sure that if there is one bracket it is marked as unbalanced
